plugins/cfu/README.md

```diff
@@ -0,0 +1,35 @@
+# Component Firmware update
+
+## Introduction
+
+CFU is a protocol from Microsoft to make it easy to install firmware on HID devices.
+
+This protocol is unique in that it requires has a pre-download phase before sending the firmware to
+the microcontroller. This is so the device can check if the firmware is required and compatible.
+CFU also requires devices to be able to transfer the entire new transfer mode in runtime mode.
+
```

Comment:
since there are some specs for this from the web, link to them in readme?

Reply:
Done.